### ISSUE TITLE
Upgrade to deno v1.10.2

### ISSUE THREAD

**Issue by jzaki** [jzaki/aggregator#9]
Supports remote import maps, plus critical security fix.

**Comment by voltrevo** [comment]
Blocked by postgres incompatibility: https://github.com/denodrivers/postgres/issues/289.

**Comment by voltrevo** [comment]
Issue is apparently fixed, just waiting for denodrivers/postgres to do a release.

Once that happens deno is further along at 1.11.0, which has updated TypeScript. This includes support for JS native private fields:

```ts
class Test {
  #test() {} // Fails to compile before deno 1.11.0
}
```

In addition to being nice-to-have, dependencies are likely to start using this syntax which means we can't use latest dependencies if we don't upgrade.

**Comment by voltrevo** [comment]
Postgres updated, just waiting on PostQuery now. PR submitted: https://github.com/DjDeveloperr/PostQuery/pull/5.
